src/main/java/com/influxdb/v3/client/InfluxDBClient.java

```diff
@@ -182,6 +182,88 @@ Stream<Object[]> query(@Nonnull final String query,
                            @Nonnull final Map<String, Object> parameters,
                            @Nonnull final QueryOptions options);
 
+    /**
+     * Query data from InfluxDB IOx using FlightSQL.
+     * <p>
+     * The result stream should be closed after use, you can use try-resource pattern to close it automatically:
+     * <pre>
+     * try (Stream&lt;Map&lt;String, Object&gt;&gt; rows = client.queryRows("select * from cpu where host=$host",
```

Comment:
The arguments in the Javadoc example here are `client.queryRows(String query, Map<String, Object> parameters, QueryOptions options)` but the documented method is `Stream<Map<String, Object>> queryRows(@Nonnull final String query)`.  The method has only 1 argument. But the example shows 3.